File: atcoder.jp/contests/ddcc2020-qual/tasks/ddcc2020_qual_a/main.py
def prize(n: int) -> int:
    return {3: 100000, 2: 200000, 1: 300000}.get(n, 0)


def additional_prize(x: int, y: int) -> int:
    if x == 1 and y == 1:
        return 400000
    return 0


X, Y = map(int, input().split())

ans = prize(X) + prize(Y) + additional_prize(X, Y)
print(ans)

File: atcoder.jp/contests/ddcc2020-qual/tasks/ddcc2020_qual_a/main.ts
import { readFileSync } from "fs";
let inputLinesIndex = 0;
const inputLines = readFileSync("/dev/stdin", "utf8").split("\n");
const input = () => inputLines[inputLinesIndex++];

const prize = (n: number) =>
  new Map([
    [3, 100000],
    [2, 200000],
    [1, 300000],
  ]).get(n) || 0;

const additionalPrize = (x: number, y: number) =>
  x === 1 && y === 1 ? 400000 : 0;

const main = () => {
  const [X, Y] = input().split(" ").map(Number);

  const ans = prize(X) + prize(Y) + additionalPrize(X, Y);
  console.log(ans);
};

main();
